src/main/resources/alma/index-config.json

```diff
@@ -39,6 +39,37 @@
           "hyphen_analyzer": {
             "tokenizer": "standard",
             "char_filter": "remove-hyphens"
+          },
+          "digibib_default": {
+            "type": "custom",
+            "tokenizer": "hyphen",
+            "filter": [
+              "lowercase",
+              "digibib_german_stop_words",
+              "keyword_repeat",
+              "icu_normalizer",
+              "icu_folding",
+              "hyphen",
+              "digibib_string_normalizer",
+              "snowball",
+              "digibib_standardnumber",
+              "unique"
```

Comment:
Here I don't know if we should take this "digibib_standardnumber" to the  already present `isbn` field.

Reply:
AFAICS, this shouldn't make a difference since you already provide ISBN-10 and ISBN-13 variants through the transformation.

Reply:
ok then